doc/nrf/ug_thingy53.rst

```diff
@@ -285,3 +262,88 @@ To build and program the source code from the command line, complete the followi
          west flash
 
    The device resets and runs the programmed sample or application.
+
+.. _thingy53_app_update:
+
+Updating firmware image
+=======================
+
+The firmware on Thingy:53 can be programmed using an external debug probe and 10-pin JTAG cable, as described in sections related to building in various environments.
+You can also update applications running on both network and application core using the built-in MCUboot bootloader and nRF Connect Programmer or the nRF Programmer for Android or iOS.
+Prebuilt application images can also be updated that way.
+See subsections below for details about updating firmware image.
+
+Firmware update using external debug probe
+------------------------------------------
+
+If you are using an external debug probe, such as the nRF5340 DK, or any J-Link device supporting ARM Cortex-M33, you do not have to use applications that follow the :ref:`thingy53_app_guide`.
+In that case you can flash the Thingy:53 in a similar way as nRF53 DK, see :ref:`ug_nrf5340` for details.
+Keep in mind that you will need to flash an sample or application compatible with Thingy:53 application guide to bring back serial recovery and DFU OTA.
+
+.. _thingy53_app_update_mcuboot:
+
+Firmware update using MCUboot bootloader
+----------------------------------------
+
+The Thingy:53-compatible samples and applications use MCUboot bootloader than can be used to update firmware.
+This method uses signed binary files :file:`app_update.bin` and :file:`net_core_app_update.bin` (or :file:`dfu_application.zip`).
+You can program the precompiled firmware image using one of the following ways:
+
+* Use the :doc:`mcuboot:index` feature and the built-in serial recovery mode of Thingy:53.
+  In this scenario Thingy is connected directly to your PC through USB.
+* Update the firmware over-the-air (OTA) using Bluetooth Low Energy and the nRF Programmer mobile application for Android or iOS.
+  To use this method, the application that is currently flashed on Thingy:53 must support it.
+  All precompiled images support OTA using Bluetooth.
+
+See `Updating Nordic Thingy:53 firmware`_ for the detailed procedures on how to program a Thingy:53 using nRF Connect Programmer or the nRF Programmer for Android or iOS.
+
+.. _thingy53_app_guide:
+
+Thingy:53 application guide
+***************************
+
+Thingy:53 does not have a built-in J-Link debug IC.
+Because of that, the samples and applications that are compatible with Thingy:53 by default use MCUboot bootloader with serial recovery support.
+Enabling DFU and MCUboot requires consistency in configuration of the samples and applications.
+
+For a  sample or application to be compatible with Thingy:53, the application must comply with the following configuration:
+
+* Partition manager configuration
```

Comment:
```suggestion
Partition manager configuration
```
This will highlight this entry. Or use a period or dash at the end of the line. Otherwise the lines will appear one after another and you'll end up with "Partition manager configuration Thingy:53 compatible samples use ..."